src/bijectors/coupling.jl

```diff
@@ -118,6 +118,20 @@ Combines `x_1`, `x_2`, and `x_3` into a single vector.
 """
 @inline combine(m::PartitionMask, x_1, x_2, x_3) = m.A_1 * x_1 .+ m.A_2 * x_2 .+ m.A_3 * x_3
 
+function ChainRulesCore.rrule(::typeof(combine), m::PartitionMask, x_1, x_2, x_3)
+    prj = map(ChainRulesCore.ProjectTo, (x_1, x_2, x_3))
```

Comment:
Can we just define three separate projectors? Then we also don't have to rely on constant propagation in the pullback:
```suggestion
    proj_x_1 = ChainRulesCore.ProjectTo(x_1)
    proj_x_2 = ChainRulesCore.ProjectTo(x_2)
    proj_x_3 = ChainRulesCore.ProjectTo(x_3)
```